src/game/items.c

```diff
@@ -791,11 +791,23 @@ void Item_TakeDamage(ITEM_INFO *item, int16_t damage, bool hit_status)
 
 bool Item_TestFrameEqual(ITEM_INFO *item, int16_t frame)
 {
-    return item->frame_number == g_Anims[item->anim_number].frame_base + frame;
+    return Item_TestAbsFrameEqual(
+        item->frame_number, g_Anims[item->anim_number].frame_base + frame);
+}
+
+bool Item_TestAbsFrameEqual(int16_t abs_frame, int16_t frame)
+{
+    return abs_frame == frame;
 }
 
 bool Item_TestFrameRange(ITEM_INFO *item, int16_t start, int16_t end)
 {
-    return item->frame_number >= g_Anims[item->anim_number].frame_base + start
-        && item->frame_number <= g_Anims[item->anim_number].frame_base + end;
+    return Item_TestAbsFrameRange(
+        item->frame_number, g_Anims[item->anim_number].frame_base + start,
+        g_Anims[item->anim_number].frame_base + end);
+}
+
+bool Item_TestAbsFrameRange(int16_t abs_frame, int16_t start, int16_t end)
+{
+    return abs_frame >= start && abs_frame <= end;
```

Comment:
Hey, since this no longer works with ITEM_INFO, I'm leaning towards creating anim.c / anim.h pair and changing the prefix of this and other functions that do not use ITEM_INFO* to Anim_. WDYT?

Reply:
Ok, I wasn't sure if we wanted to make new `anim.h/c` files that would only include these two functions. I can do that.